tracks/scala/exercises/perfect-numbers/mentoring.md

```diff
@@ -0,0 +1,59 @@
+### Reasonable solutions
+
+#### Solution using case objects
+
+```scala
+object PerfectNumbers {
+  import NumberType._
+
+  def classify(number: Int): NumberType = {
```

Comment:
The return type is `NumberType`, whereas the next solution's `classify` method's return type is `: Either[String, NumberType.Value]`. Is that correct?

Reply:
Good catch. I pulled an old solution as an example that needs to be updated to the new signature/unit tests.